cms/settings/local.py.example

```diff
@@ -1,8 +1,10 @@
-CACHES = {
-    'default': {
-        'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
-    }
-}
+# uncomment the following lines if running outside of the Docker container
+# --- outside Docker ---
+# import dj_database_url
+
+# DATABASES = {"default": dj_database_url.config(default="sqlite:///cms.sqlite3")}
+# CACHES["default"]["BACKEND"] = "django.core.cache.backends.dummy.DummyCache"
```

Comment:
Issue: No cache at all is going to have very different performance (and possibly behaviour) characteristics. I'd suggest using `LocMemCache` instead, as it'll at least still function as a cache.
```suggestion
# CACHES["default"]["BACKEND"] = "django.core.cache.backends.locmem.LocMemCache"
```